config/config.go

```diff
@@ -275,6 +283,10 @@ func NewAgentConfig(agentIni *File, agentYaml *YamlAgentConfig) (*AgentConfig, e
 		cfg.ContainerBlacklist = agentIni.GetStrArrayDefault(ns, "container_blacklist", ",", cfg.ContainerBlacklist)
 		cfg.ContainerWhitelist = agentIni.GetStrArrayDefault(ns, "container_whitelist", ",", cfg.ContainerWhitelist)
 		cfg.ContainerCacheDuration = agentIni.GetDurationDefault(ns, "container_cache_duration", time.Second, 30*time.Second)
+
+		// windows args config
+		cfg.WindowsProcessRefreshInterval = agentIni.GetIntDefault(ns, "windows_refresh_interval", cfg.WindowsProcessRefreshInterval)
```

Comment:
Since we do `n % cfg.WindowsProcessRefreshInterval` we need to make sure this is not `0` to avoid a divide by zero error